packages/fxa-settings/src/components/AlertBar/index.tsx

```diff
@@ -2,47 +2,70 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import React, { ReactNode } from 'react';
+import React, { ReactNode, useContext } from 'react';
+import { createPortal } from 'react-dom';
+import AlertBarContext from '../../lib/AlertBarContext';
```

Comment:
Nitpick - sometimes the module dependency is used (`fxa-settings/src/lib/AlertBarContext`), sometimes the relative path is used. I'm fine with either, but ideally we could be consistent.

Reply:
Ah good catch, this was probably from some auto import update thing I didn't notice since I'd originally named this something else.